app/screens/select_server/select_server.js

```diff
@@ -212,7 +274,20 @@ export default class SelectServer extends PureComponent {
             return;
         }
 
-        this.pingServer(url);
+        if (LocalConfig.ExperimentalClientSideCertEnable && Platform.OS === 'ios') {
```

Comment:
Do we need to check this if true? Like:
``LocalConfig.ExperimentalClientSideCertEnable === 'true'``

Reply:
this is actually a boolean `true`|`false` not a string, the local config is set in the assets config.json file

Reply:
Ah, right, I got it mixed up with `config.ExperimentalClientSideCertEnable`